cmd/jwt_test.go

```diff
@@ -24,7 +24,8 @@ func testAuthenticate(authType string, t *testing.T) {
 		t.Fatalf("unable initialize config file, %s", err)
 	}
 	defer removeAll(testPath)
-
+	// Create access and secret keys in length, 300 and 600
+	getNewCredential(300, 600)
```

Comment:
As you don't keep the return values, I am not sure what exactly you are trying to do here.